contracts/template/Cargo.toml

```diff
@@ -1,25 +1,16 @@
 [package]
```

Comment:
I like how these dependency changes prevent us from having to update the dependencies all the time. I think we might want to think through this contract a little bit.

I'm not sure what the best solution is here, but we may want to consider how people obtain and use the template. If we mark the dependencies as workspace = true, then will they still be able to just grab the template and have it working out of the box?

This is just one example, but maybe we want some type of tool or script to grab the template? It could replace workspace = true with the actual dependencies from the root.

Reply:
Excellent point. My initial thoughts are to either put the template back to managing it's own dependencies for now and then later move it to it's own repo and refactor so it can be used like the [cosmwasm template](https://github.com/CosmWasm/cw-template), thoughts?

Reply:
Sure, that works for me.

Reply:
fixed